llama_toolchain/agentic_system/client.py

```diff
@@ -134,8 +138,11 @@ async def run_main(host: str, port: int):
     api = AgenticSystemClient(f"http://{host}:{port}")
 
     tool_definitions = [
-        SearchToolDefinition(engine=SearchEngineType.bing),
-        WolframAlphaToolDefinition(),
+        SearchToolDefinition(
```

Comment:
Can we make this example be for Brave (for people who are used to the current config having Brave keys)?